src/helpers/issue.ts

```diff
@@ -281,14 +281,15 @@ export const closePullRequest = async (pull_number: number) => {
   const payload = context.payload as Payload;
   const logger = getLogger();
   try {
-    await getBotContext().octokit.rest.pulls.update({
+    return await getBotContext().octokit.rest.pulls.update({
```

Comment:
It is unclear to me what the return value is of this function from this code review view.

Reply:
I have updated code, please check.